design/async-commit/spec.md

```diff
@@ -68,15 +73,21 @@ message LockInfo {
 
 Implemented in TiDB, TiSpark.
 
-The client has system variables for supporting async commit (`tidb_enable_async_commit`), supporting 1pc (`tidb_enable_1pc`), and forcing external consistency when using async commit (`tidb_guarantee_external_consistency`). The client has configuration options for setting the limits on the size (`tikv-client.async-commit.total-key-size-limit`) and number of keys (`tikv-client.async-commit.keys-limit`) in an async commit transaction. When async commit is enabled, a transaction will not use async commit if it exceeds either of the size limits.
+The client has system variables for supporting async commit (`tidb_enable_async_commit`), supporting 1pc (`tidb_enable_1pc`), and forcing external consistency when using async commit (`tidb_guarantee_external_consistency`). The client has configuration options for setting the limits on the size (`tikv-client.async-commit.total-key-size-limit`) and number of keys (`tikv-client.async-commit.keys-limit`) in an async commit transaction. When async commit is enabled, a transaction will not use async commit if it exceeds either of the size limits. The rest of the spec assumes that async commit or 1pc are enabled.
+
+The client must set `use_async_commit` and/or `try_one_pc` in the `PrewriteRequest`. If using async commit, then the `secondaries` field should contain the keys of all keys locked in the transaction except the primary key. If the user wants external consistency, a new timestamp is fetched from PD and used as the `min_commit_ts`. Otherwise, it is set to the maximum of the transaction's start and 'for update' timestamps (if we get a timestamp from PD it should be greater than both of those).
```

Comment:
min_commit_ts needs an extra 1. So it won't be equal to start ts or for update ts.